Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Settings.
+ARG USER_ID=3001
+ARG USER_NAME=gifmbutton-service
```

Comment:
Why are we putting service in the username? Can we just use gifm or something simple?